src/test/java/com/lgtm/easymoney/controllers/TransferControllerTest.java

```diff
@@ -190,11 +195,10 @@ public void transferFailedWithEmptyCategory() throws Exception {
   @Test
   public void getTransfersSuccess() throws Exception {
     // Arrange
-    Mockito.when(transferService.getTransfersByUid(fromUid)).thenReturn(transferRsp);
+    Mockito.when(transferService.getTransfers(fromUser)).thenReturn(transferRsp);
```

Comment:
does all services have this change? input param changes from uid to user? im guessing this is because of the added auth

Reply:
yes